docs/index.html

```diff
@@ -25,7 +21,7 @@
 
     */
 
-    var content = { 
+    const content = { 
         defaultImage: "http://i.giphy.com/l0HlRnAWXxn0MhKLK.gif",
```

Comment:
ah. yeah, i wasn't sure if you still needed/wanted that for something. okay, gonna remove it.